sdlf-ddk-lightweight/python/requirements.txt

```diff
@@ -1,8 +1,8 @@
-aws_ddk_core==1.0.1
-boto3==1.24.11
-aws-cdk-lib==2.71.0
-aws-cdk.aws-glue-alpha==2.71.0a0
-aws-cdk.aws-kinesisfirehose-alpha==2.71.0a0
-aws-cdk.aws-kinesisfirehose-destinations-alpha==2.71.0a0
+aws_ddk_core==1.1.0
```

Comment:
Can we remove the version pin to use latest, just tested with `1.3.1`

Reply:
Hi, 

I have updated the pipeline synth action in app.py to pass ec2:DescribeAvailbilityZones to the synth stage. I had tested it by passing it as a flag in cdk.json context but did not push that as I was not sure. 

Regarding the aws_ddk_core version , I have changed that to 1.3.1. Please let me if I should remove it all together.